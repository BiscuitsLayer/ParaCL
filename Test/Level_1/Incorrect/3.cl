temp = 5;
temp = 0;
if if (temp + (7 / 3) * (5 + (11 / 1) / ?)) {
    print 1000;
}
print 100;