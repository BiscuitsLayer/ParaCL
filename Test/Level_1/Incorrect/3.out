Syntax error!
Line: 3, Columns: 4 - 6:
if if ...